pysb/simulator/scipyode.py

```diff
@@ -726,3 +676,17 @@ def jac_fn(t, y, p):
         trajectory[i:, :] = 'nan'
 
     return trajectory
+
+
+class DummyExecutor(Executor):
```

Comment:
Nice. I can't believe this isn't part of the standard `concurrent.futures` module. Nitpick: It should be called something like `SerialExecutor`, as "Dummy" would imply to me that it doesn't actually run anything.